modules/ROOT/pages/administration/databases.adoc

```diff
@@ -1062,7 +1198,8 @@ SHOW DATABASES YIELD name
 | +"sci-fi"+
 | +"system"+
 | +"topology-example"+
-13+d|Rows: 8
+| +"options-example"+
```

Comment:
I think this list is supposed to be ordered by name so the options one should probably be higher up (even if the docs tests don't check that)